bugbug/model.py

```diff
@@ -254,8 +255,21 @@ def train(self, importance_cutoff=0.15):
 
         print(f"No confidence threshold - {len(y_test)} classified")
         confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=class_names)
-        print(confusion_matrix)
         tracking_metrics["confusion_matrix"] = confusion_matrix.tolist()
+        confusion_matrix_reformatted = tracking_metrics["confusion_matrix"].copy()
+        class_names_string = [str(item) for item in class_names]
```

Comment:
You don't really need this, you can use f-strings later to handle the conversion

Reply:
E.g.:
```Python
f"{class_name} (Predicted)"
```

Reply:
f-strings produce results like:
`f"{class_names}"` outputs `'[0, 1]'` instead of `['0', '1']` .

This is how I can still make it work
`        for i in range(len(confusion_matrix_table)):
            confusion_matrix_header.append(f"{class_names[i]} (Predicted)")`

Let me know if that's okay

Reply:
Of course, that was just an example. f-string is just a way to format a variable, if you pass the list you will format the list, if you pass an element of the list you will format the element of the list.